stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/model/AnnotatedClass.java

```diff
@@ -67,7 +66,7 @@
 
         FieldOption fieldOption = useStag != null ? useStag.value() : null;
         if (fieldOption == null) {
-            useStag = ElementUtils.findAnnotation(UseStag.class, element);
+            useStag = element.getAnnotation(UseStag.class);
```

Comment:
I think it makes sense to remove.  Technically it could still be supported (i.e., a lack of FieldOptions specification could cause a search through parent class annotations) but I think that would be surprising behavior to the consumer.  👍 to remove from me.